### ISSUE TITLE
Mistake in codes

### ISSUE THREAD

**Issue by jaydev0220** [Bananamilk452/better-ytsr#1]
At index.js line 20:
`key = key.toLowerCase()`

this causes error when sortBy and uploadDate filters are used

**Comment by Bananamilk452** [comment]
Thank you for bug report! I will check code and fix in few hours.

**Comment by Bananamilk452** [comment]
+ can you show your code to know how error is caused?

**Comment by jaydev0220** [comment]
my codes
```
const ytsr = require("better-ytsr");

async function search(query) {
  const res = await ytsr(query, {
    filters: {
      sortBy: "Relevance"
    }
  });

  console.log(res);
}

search("ncs");
```

error
```
      const value = option[key].toLowerCase();
                                ^

TypeError: Cannot read properties of undefined (reading 'toLowerCase')
    at parseSP (C:\Users\jay95\Desktop\Projects\Bots\Musify\node_modules\better-ytsr\lib\index.js:28:33)
    at main (C:\Users\jay95\Desktop\Projects\Bots\Musify\node_modules\better-ytsr\lib\index.js:44:47)
    at search (C:\Users\jay95\Desktop\Projects\Bots\Musify\test.js:4:21)
    at Object.<anonymous> (C:\Users\jay95\Desktop\Projects\Bots\Musify\test.js:13:1)
    at Module._compile (node:internal/modules/cjs/loader:1095:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1147:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47
```

**Comment by Bananamilk452** [comment]
fixed in 1.1.1. please update to latest version,

[npm](https://www.npmjs.com/package/better-ytsr)
